server/channels/app/platform/license.go

```diff
@@ -104,6 +104,13 @@ func (ps *PlatformService) LoadLicense() {
 	if nErr != nil {
 		if ps.Config().FeatureFlags.EnableMattermostEntry && model.BuildEnterpriseReady == "true" {
 			ps.logger.Info("Mattermost Entry is enabled. Unlocking enterprise features.")
+
+			if ps.LicenseManager() == nil {
+				ps.logger.Warn("License manager not available, setting license to nil.")
+				ps.SetLicense(nil)
+				return
+			}
```

Comment:
@nickmisasi had to make this change here because this test was failing with the FF on: https://github.com/mattermost/mattermost/blob/a25bf4180146a8c4572a8e6f467b087b3e195b6f/server/channels/app/plugin_api_test.go#L2252

The test was forcing a `BuildEnterpriseReady` to `true` even though the test environment does not have the enterprise interfaces registered so when reaching this point to set the entry license `LicenseManager` would panic